CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 
     set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} /GL /Zi")
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /GL /Zi")
-    set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /LTCG /DEBUG /OPT:REF /OPT:ICF")
+    set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /LTCG /USERPROFILE:PGD=${CMAKE_CURRENT_SOURCE_DIR}/src/bin/winuser/msquic.pgd /DEBUG /OPT:REF /OPT:ICF")
```

Comment:
does `/DEBUG` belong in the release flags?

Reply:
The `/DEBUG` flag just causes `.pdb` files to be generated.